roles/openshift_registry/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- set_fact: _option_images="--images={{ openshift_registry_url|quote }}"
```

Comment:
We aren't doing the _ anymore, however maybe we should. Hmm.

This would more likely be something like this:
```
- set_fact: oreg_images="--images={{ oreg_url|quote }}"
```